src/test/java/ca/corefacility/bioinformatics/irida/model/sample/unit/SampleTest.java

```diff
@@ -0,0 +1,127 @@
+package ca.corefacility.bioinformatics.irida.model.sample.unit;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Map;
+import java.util.UUID;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+
+import ca.corefacility.bioinformatics.irida.model.sample.MetadataTemplateField;
+import ca.corefacility.bioinformatics.irida.model.sample.Sample;
+import ca.corefacility.bioinformatics.irida.model.sample.metadata.MetadataEntry;
+import ca.corefacility.bioinformatics.irida.model.sample.metadata.PipelineProvidedMetadataEntry;
+import ca.corefacility.bioinformatics.irida.model.sequenceFile.SequenceFilePair;
+import ca.corefacility.bioinformatics.irida.model.workflow.submission.AnalysisSubmission;
+
+public class SampleTest {
```

Comment:
There's already a `ca.corefacility.bioinformatics.irida.model.SampleTest`.  Merge these methods into that class please.

Reply:
Somehow I must have missed that. Merged in 9987e4806b0996665cca2ca04e2cae2fc0d322fb